Tests/DatadogTests/Datadog/Mocks/RUMFeatureMocks.swift

```diff
@@ -88,43 +88,38 @@ extension RUMResourceType {
 
 // MARK: - RUMDataModel Mocks
 
-struct RUMDataModelMock: RUMDataModel, Equatable {
+struct RUMDataModelMock: RUMDataModel, RUMSanitizableEvent, EquatableInTests {
     let attribute: String
+    var usr: RUMUser?
+    var context: RUMEventAttributes?
 }
 
 // MARK: - Component Mocks
 
 extension RUMEvent: AnyMockable where DM == RUMViewEvent {
     static func mockAny() -> RUMEvent<RUMViewEvent> {
-        return .mockWith(model: RUMViewEvent.mockRandom())
+        return RUMEvent(model: RUMViewEvent.mockRandom())
     }
 }
 
 extension RUMEvent {
-    static func mockWith<DM: RUMDataModel>(
-        model: DM,
-        attributes: [String: Encodable] = [:],
-        userInfoAttributes: [String: Encodable] = [:]
-    ) -> RUMEvent<DM> {
-        return RUMEvent<DM>(
-            model: model,
-            attributes: attributes,
-            userInfoAttributes: userInfoAttributes
-        )
-    }
-
     static func mockRandomWith<DM: RUMDataModel>(model: DM) -> RUMEvent<DM> {
```

Comment:
I think now this function is not needed at all. All `RUMDataModels` already implement the `mockRandom()` function in `RUMDataModelMocks.swift` and it produces random `usr` and `context`. WDYT?

Reply:
I agree, good catch!